docs/20_smart-contracts/40_guides/10_create_smart_contracts_guide.md

```diff
@@ -0,0 +1,1351 @@
+
+### Create, Build and Deploy a DUNE Bare Application
+
+The bare application contains just the minimum files needed to develop, compile and deploy one smart contract to the blockchain. Run the following command to create a bare `hello` application.
+
+```shell
+dune --create-bare-app hello .
+cd hello
+ls
+```
+
+The result of this command is a `hello` directory with four files in it:
+
+- README.txt, a text file which contains information about how to build this application.
+- hello.contracts.md, the ricardian contract markdown file.
+- hello.hpp, the C++ header file.
+- hello.cpp, the C++ implementation file.
+
+#### Build the DUNE Bare Application
+
+To build the DUNE bare application run the following command:
+
+```shell
+dune -- cdt-cpp /host/<LOCAL-PATH-TO-HELLO-PARENT>/hello/hello.cpp -o /host/<LOCAL-PATH-TO-HELLO-PARENT>/hello/hello.wasm
+```
+
+The above command you can run it from directory on your local machine, just make sure that the `<LOCAL-PATH-TO-HELLO-PARENT>` is replace with the corresponding path of the parent directory of the hello folder.
+
+The result of the above build command are two files:
+
+- hello.wasm, the WebAssembly binary file for the smart contract.
+- hello.abi, the application binary interface (ABI) file for the smart contract.
+
+#### Deploy the hello Smart Contract (built with cdt-cpp)
+
+Execute the following command to deploy the `hello` smart contract to the account `hello` which you created at the beginning:
+
+```shell
+dune --deploy <PATH_TO_HELLO_DIR>
+```
+
+### Create, Build and Deploy a DUNE cmake Application
+
+The DUNE cmake application is a more complex type than the bare type and is suited better for larger projects which involve multiple contracts and/or web applications.
+
+```shell
+dune --create-cmake-app hello .
+cd hello
+ls
+```
+
+The result of this command is a `hello` directory with a bit more complex structure than the bare application because it generates some extra files to configure the cmake project:
+
+- CMakeLists.txt, cmake configuration file.
+- README.txt, a text file which contains information about how to build this application with cmake.
+- build, the output build folder, at the beginning is empty.
+- include, the C++ include files folder, at the beginning it contains just hello.hpp file.
+- ricardian, the folder contains the smart contract ricardian definition, hello.contracts.md file.
+- src, the C++ implementation files folder, at the beginning it contains just hello.cpp file.
+
+#### Build the DUNE cmake Application
+
+To build the DUNE cmake application run the following command:
+
+```shell
+dune --cmake-build <PATH_TO_CMakeLists.txt_PARENT_DIR>
+ls <PATH_TO_CMakeLists.txt_PARENT_DIR>/build/hello
+```
+
+You can run the two commands from above from any directory on your local machine, just make sure you correctly replace the `<PATH_TO_CMakeLists.txt_PARENT_DIR>` with the parent directory path of `CMakeLists.txt` file.
```

Comment:
Same here, command is adequately self-explanatory. Text below doesn't offer any extra information.